test/src/406-ducc-webhook-notifications/main

```diff
@@ -0,0 +1,96 @@
+cvmfs_test_name="Webhook notifications test"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="ducc"
+
+cleanup() {
+  pid=$(ps aux | grep 'registry-webhook.py' | awk -F' ' '{print $2}' )
+  kill $pid
+}
+
+check_port() {
+    if nc -zv 127.0.0.1 8080 2>&1;
+    then
+        return 0
+    else
+        check_port
+    fi
+}
+
+check_id_action() {
+    expected_id=$1
+    expected_action=$2
+
+    notification=$( tail -n 1 $notifications_file )
+    id=(${notification//|/ })
+
+    if [ ${id[0]} != ${expected_id} ]
+    then
+        echo 'The action id is not correct' && return 10
```

Comment:
Here and in the next check, it would be good to report what was expected and what was found.